test/Microsoft.Extensions.Primitives.Tests/MockChangeToken.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Primitives.Tests
+{
+    public class MockChangeToken : IChangeToken
```

Comment:
`List<Tuple<Action<object>, object, MockDisposable>>` looks super ugly. If we can avoid having that, I'd be all for it.